modules/custom_operations/pyproject.toml

```diff
@@ -38,3 +46,16 @@ line-length = 119
 
 [tool.ruff.isort]
 lines-after-imports = 2
+
+[tool.scikit-build]
+cmake.minimum-version = "3.15"
+cmake.build-type = "Release"
+cmake.args = ["-DCUSTOM_OPERATIONS:STRING=tokenizer", "-DBUILD_FAST_TOKENIZERS=OFF"]
+wheel.packages = ["user_ie_extensions/tokenizer/python/ov_tokenizer"]
+wheel.install-dir = "ov_tokenizer/libs"
+wheel.py-api = "py3"
+sdist.cmake = true
```

Comment:
I suppose we also need to set:
- wheel.license-files to add licenses
- sdist.exclude = [] to exclude tests
- experimental = True to mark as experimental